h2o-py/h2o/model/model_base.py

```diff
@@ -1376,9 +1376,9 @@ def __plot_1d_pdp_multinomial(self, col, i, data, pps, data_start_index, fig, gx
     # note that, x stays at one value for the duration of y value changes.
     def __pred_for_3d(self, x, y, z, colPairs, nbins, user_cols, user_num_splits):
         # deal with y axis first
-        np = _get_numpy("2D partial plots")
-        if np is None:
-            print("Numpy not found.  Cannot plot 2D partial plots.")
+        if not can_use_numpy():
+            raise ImportError("numpy is required for 2D partial plots.")
```

Comment:
2D or 3D?

Reply:
Yeah i guess the comment was wrong.  I updated to 3D.  Thanks.